src/main/resources/application.conf

```diff
@@ -2,4 +2,17 @@ latis {
   services = [
     {type: "class", mapping: "/hapi", class: "latis.service.hapi.HapiService"}
   ]
+
+  hapi {
+    about {
+      // Environment variable overrides
+      id = ${?FORCED_ID}
+      title = ${?FORCED_TITLE}
+      contact = ${?FORCED_CONTACT}
+      description = ${?FORCED_DESCRIPTION}
+      contactId = ${?FORCED_CONTACTID}
+      citation = ${?FORCED_CITATION}
+    }
+  }
```

Comment:
I think these would be better in `reference.conf` because we're saying the default value is taken from an environment variable.

We've been deriving the environment variable names by upper-casing everything and replacing periods with underscores, like `LATIS_HAPI_ABOUT`.

Reply:
Okay.